omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx.conf.erb

```diff
@@ -148,6 +150,13 @@ http {
             more_clear_headers Server;
             access_log /var/log/opscode/nginx/rewrite-port-80.log;
             return 301 https://$host$request_uri;
+            location @errorrespfilter {
+              return 301 https://$host$request_uri;
+              header_filter_by_lua_block { ngx.header.content_length = nil }
+              body_filter_by_lua '
+              ngx.arg[1] = ngx.re.sub(ngx.arg[1],"openresty", "chef")
```

Comment:
Should this be changed to ngx.arg[1] = ngx.re.sub(ngx.arg[1],"openresty", "") or some non consequential string instead?

Reply:
Agree, if the idea is that we don't want to reveal anything about the system from the error.